pajbot/apiwrappers/twitch/helix.py

```diff
@@ -726,7 +726,30 @@ def fetch_all_moderators(self, broadcaster_id, authorization):
         """Calls the _fetch_moderators_page function using the broadcaster_id parameter.
         broadcaster_id is a required field and must match the user ID in authorization."""
         moderator_ids = self._fetch_all_pages(self._fetch_moderators_page, broadcaster_id, authorization)
+    def _fetch_vips_page(
+        self,
+        broadcaster_id: str,
+        authorization,
+        after_pagination_cursor=None,
+    ):
+        """Calls the Get VIPs Helix endpoint using the broadcaster_id parameter.
+        broadcaster_id is a required field and must match the user ID in authorization."""
+        response = self.get(
+            "/channels/vips",
+            {"broadcaster_id": broadcaster_id, "first": 100, **self._with_pagination(after_pagination_cursor)},
+            authorization=authorization,
+        )
+
+        vips = [UserBasics(entry["user_id"], entry["user_login"], entry["user_name"]) for entry in response["data"]]
+        pagination_cursor = response["pagination"].get("cursor", None)
+
+        return vips, pagination_cursor
+
+    def fetch_all_vips(self, broadcaster_id: str, authorization):
+        """Calls the _fetch_vips_page function using the broadcaster_id parameter.
+        broadcaster_id is a required field and must match the user ID in authorization."""
+        vips = self._fetch_all_pages(self._fetch_vips_page, broadcaster_id, authorization)
 
-        moderator_ids = list(set(moderator_ids))
```

Comment:
What‘s happening here, why is this being deleted/replaced?

Reply:
Something fucked up in the merge process. I'll merge the other prs in first, which should hopefully fix this one up.